### DISCUSSION TITLE
Mevo Plus Limiting Speed

### DISCUSSION THREAD

**Discussion by Wrightn13** [springbok/MLM2PRO-GSPro-Connector#109] [General]
Hello,  is there some kind of ball and club speed max in the connector? im doing some long drive training and have no issues with the flightscope numbers but for some reason it seems whenever i get over 200 balls speed or 140ish club the numbers dont transfer from flightscope to GSPro. it just reads as like 20 mph ball and 15 mph club and the ball just dribbles off the tee. is there some type of cap on the Connector thats causing this?

**Comment by springbok** [comment]
It will auto correct a ball speed over 200, a club speed over 140, or spin rate > 13000 by dividing the number by 10. This was done as anything higher than these number were deemed to be a misread due to OCR errors.

**Reply by Wrightn13** [reply]
@springbok gotcha. is there any way to change these limits? if not i can figure something out but it would be nice to keep using GSpro even during some speed training sessions?

**Reply by springbok** [reply]
They can be adjusted but at the time those limits seemed reasonable for most golfers using the connector. What limits would you like to see?

**Reply by Wrightn13** [reply]
@springbok i completely understand its not the norm to see those numbers. ive been up to 150 club speed and around 220 ball before so giving some room for increased speed maybe 160 club and like 240 ball. i dont think i would ever reach those numbers. i understand this is a weird case and if it is to much work to get those increased its no big deal. thank you.

**Comment by wonder99** [comment]
That's unbelievable--I'd be happy with 150 ball speed!  Please share a
video!! :)

**Reply by Wrightn13** [reply]
![image](https://github.com/user-attachments/assets/a6dee06b-f453-4ed2-8fa8-6c9011f6d3cd)

**Reply by Wrightn13** [reply]
i dont have any swing videos right now but here is a screenshot of a session i was just working on club head speed here and didnt care much about the strike with how bad the smash factor is

**Comment by wonder99** [comment]
That's crazy fast!  Do you compete?

**Reply by Wrightn13** [reply]
@wonder99 no not yet. im thinking about possibly getting into it next year
